main.go

```diff
@@ -246,3 +256,52 @@ func bindFlags(cmd *cobra.Command, v *viper.Viper) {
 		}
 	})
 }
+
+// Create an issue in the provider repo with a message describing the upgrade failure
+func createFailureIssue(ctx upgrade.Context, repoOrg string, repoName string, errMsg string) (string, error) {
+	now := time.Now()
+	y, m, d := now.Date()
+	title := fmt.Sprintf("Upgrade provider failure: %v %v %v", y, m, d)
+
+	getIssues := exec.CommandContext(ctx, "gh", "search", "issues",
+		"Upgrade provider failure: ",
+		"--repo="+repoOrg+"/"+repoName,
```

Comment:
Can or should we also search for author here?

Reply:
We can assume `pulumi-bot` so we are updated on automation failures – will add this later after testing